tools/askor/askor_de.xml

```diff
@@ -0,0 +1,128 @@
+      #end if
+      #if ${thresholds.th_cpm}
+        --th_cpm ${thresholds.th_cpm}
+      #end if
+      #if ${thresholds.rep}
+        --rep ${thresholds.rep}
+      #end if
+      #if ${thresholds.fdr}
+        --th_FDR ${thresholds.fdr}
+      #end if
+      #if ${stats.norm}
+        --normalization ${stats.norm}
+      #end if
+      #if ${stats.adj}
+        --adj ${stats.adj}
+      #end if
+      #if ${stats.glm}
+        --glm ${stats.glm}
+      #end if
+      #if ${adv.nh}
+        --nh ${adv.nh}
+      #end if
+    ]]></command>
+    <inputs>
+        <param format="tsv" name="fileofcounts" type="data" label="File of counts" />
+        <param format="tsv" name="samples" type="data" label="descriptions of the samples" />
+        <param format="tsv" name="contrasts" type="data" label="descriptions of the contrasts" />
+        <section name="adv" title="Format options" expanded="false">
+          <param name="colgenes" type="integer" label="Column of the genes" value="1" optional="true" />
+          <param name="sep" type="text" label="field separator" value="\t" optional="true" />
+          <param name="palette" type="text" label="color palette (Rcolorbrewer)" value="Set2" optional="true" />
+          <param name="organism" type="text" label="organism" value="Asko" optional="true" />
+          <param name="nh" type="integer" label="number of genes in the heatmap" value="50" optional="true" />
+        </section>
+        <section name="selection" title="Samples selection options" expanded="false">
+          <param name="sel" type="text" label="list of selected samples (separated by comma)" value="" optional="true" />
+          <param name="remove" type="text" label="list of removed samples (separated by comma)" value="" optional="true" />
+        </section>
+        <section name="thresholds" title="Threshold options" expanded="false">
+            <param name="th_cpm" type="float" label="minimum CPM value for selecting a gene" value="0.5" optional="true" />
+            <param name="rep" type="integer" label="number of samples with CPM > th_comp for selecting a gene" value="3" optional="true" />
+            <param name="th_FDR" type="float" label="FDR threshold for reporting a differential gene" value="0.05" optional="true" />
+        </section>
+        <section name="stats" title="Statistics options" expanded="false">
+          <param name="norm" type="select" label="normalization method" value="TMM">
+            <option value="TMM">TMM</option>
+            <option value="RLE">RLE</option>
+            <option value="upperquartile">upperquartile</option>
+            <option value="upperquartile">none</option>
+          </param>
+          <param name="GLM" type="select" label="GLM method" value="qlf">
+            <option value="qlf">qlf</option>
+            <option value="lrt">lrt</option>
+          </param>
+          <param name="adj" type="select" label="Multitest correction method" value="fdr">
```

Comment:
No value attribute in a select : 

https://docs.galaxyproject.org/en/latest/dev/schema.html#select